### PR TITLE
Minor rewording of precommit docs

### PR DESCRIPTION
**Pull request by ivirshup**
Collapses ruff checks into other pre commit checks.

Unifies section on how to modify pre-commit checks.

### CODE REVIEW

File: {{cookiecutter.project_name}}/docs/template_usage.md [hunk 1]
```diff
@@ -150,10 +154,20 @@ The following pre-commit hooks are for code style and format:
     black on Python code in docs.
 -   [prettier](https://prettier.io/docs/en/index.html):
     standard code formatter for non-Python files (e.g. YAML).
+-   [ruff][] based checks:
+    -   [isort](https://beta.ruff.rs/docs/rules/#isort-i) (`I`):
```
Comment (flying-sheep) [inline {{cookiecutter.project_name}}/docs/template_usage.md:158]: When read from top to bottom, a user has no idea what the `I` means here. There should be some kind of pointer before this line.

File: {{cookiecutter.project_name}}/docs/template_usage.md [hunk 2]
```diff
@@ -226,7 +211,7 @@ The `RUF100` check will remove `noqa` IDs that are no longer necessary.
 If you want to add an ID that comes from a tool other than Ruff,
 add it to Ruff’s [`external = [...]`](https://beta.ruff.rs/docs/settings/#external) setting to prevent `RUF100` from removing it.
 
-You can also disable certain error messages for the entire project.
+Alternatively, you can disable certain error messages for the entire project.
```
Comment (flying-sheep) [inline {{cookiecutter.project_name}}/docs/template_usage.md:214]: I removed the “alternatively” beacuse Iafter the `RUF100` paragraph, I thought it’s no longer clear that it refers to the paragraph that’s now two ago.

**Review by flying-sheep — CHANGES_REQUESTED** [review]

File: {{cookiecutter.project_name}}/docs/template_usage.md (hunk 2 above)
Comment (ivirshup) [reply {{cookiecutter.project_name}}/docs/template_usage.md]: I've rewritten this section.

File: {{cookiecutter.project_name}}/docs/template_usage.md (hunk 1 above)
Comment (ivirshup) [reply {{cookiecutter.project_name}}/docs/template_usage.md]: I've added some context, but maybe we should flip the order of "configuration" and "default checks" sections?

**Comment by ivirshup** [comment]
Rewrote some of the relevant sections.

Test failure seems to be codecov on the instance repo?

**Comment by grst** [comment]
The ruff link renders incorrectly
image

File: {{cookiecutter.project_name}}/docs/template_usage.md [hunk 3]
```diff
@@ -150,10 +154,20 @@ The following pre-commit hooks are for code style and format:
     black on Python code in docs.
 -   [prettier](https://prettier.io/docs/en/index.html):
     standard code formatter for non-Python files (e.g. YAML).
+-   [ruff][] based checks:
+    -   [isort](https://beta.ruff.rs/docs/rules/#isort-i) (error code: `I`):
```
Comment (flying-sheep) [inline {{cookiecutter.project_name}}/docs/template_usage.md:158]: For consistency, we should use Ruff terminology:

grafik

- `E`, `F`, …: “rule category” or “rule prefixes”

  grafik

- `E402`, …: “rule code”

  grafik

**Review by flying-sheep — CHANGES_REQUESTED** [review]

File: {{cookiecutter.project_name}}/docs/template_usage.md (hunk 3 above)
Comment (flying-sheep) [reply {{cookiecutter.project_name}}/docs/template_usage.md]: I fixed it. Leaving this unresolved for acknowledgement

File: {{cookiecutter.project_name}}/docs/template_usage.md (hunk 1 above)
Comment (flying-sheep) [reply {{cookiecutter.project_name}}/docs/template_usage.md]: Looks good, only the terminology is a bit off, see https://github.com/scverse/cookiecutter-scverse/pull/151#discussion_r1115420970

**Review by grst — APPROVED** [review]
LGTM

**Comment by grst** [comment]
@flying-sheep, are you happy with this now?

**Review by flying-sheep — APPROVED** [review]
I’m happy! just wanted to make sure everyone is aware of the changed terminology

**Comment by ivirshup** [comment]
Thanks for cleaning this up @grst and @flying-sheep!

**Pull request merged**